masschallenge/django-accelerator
# AC-8169 :: Create Minimal Expert Navigation Tree and Landing Page

**Changes introduced in [AC-8169](https://masschallenge.atlassian.net/browse/AC-8169)**
- Add user_type to the nav tree items to enable expert only side and sub-nav items
- Add a redirect to expert overview landing page. `/dashboard/expert/overview/`

**How to test**
Checkout this PR and the sibling [PR](https://github.com/masschallenge/django-accelerator/pull/348)
**Test the redirect**
- Create a new fluent page [`Expert`](http://localhost:8181/admin/fluent_pages/page/add/) and publish it as a `category header` as shown below:
<img width="1337" alt="Screen Shot 2020-09-29 at 3 27 15 PM" src="https://user-images.githubusercontent.com/16463335/94606486-8b308800-0268-11eb-820d-6e79fe136722.png">
Create another fluent page [`overview`](http://localhost:8181/admin/fluent_pages/page/add/) and publish it as a `page` as shown below:
<img width="1333" alt="Screen Shot 2020-09-29 at 3 33 36 PM" src="https://user-images.githubusercontent.com/16463335/94606972-45c08a80-0269-11eb-8482-5130cedc4266.png">
- After the Category header and page have been created, navigate to the [fluent page admin](http://localhost:8181/admin/fluent_pages/) drag and drop the  overview page on the the just created Category header.

- Masquerade as an expert and see that you are redirected to `/dashboard/expert/overview/`


**Test that side nav items can be displayed according to user_type**

To ensure that the side nav items can be displayed according to user_type
- As demoadmin, go to this side nav item http://localhost:8181/admin/mc/navtree/1/change/item_82/ and set the user_type to Entrepreneur
- Masquerade as this  [expert](http://localhost:8181/admin/simpleuser/user/56778/masquerade/) user 
- See that on the side nav the item isn't displayed.
